devel/gk/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gk
+version                 1.0.6
+
+categories              devel
+platforms               darwin
+license                 CC-BY-3.0-US
```

Comment:
You can see a list of valid values for the `license` field here: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

Looks like this may be `CCBY-3`

Reply:
Thank you, I couldn't find that list before. I'll make the changes